semana-02/todo/add.py

```diff
@@ -0,0 +1,43 @@
+import json
+import uuid
+
+from file_manager import write_file, read_file
+
+def validate_task_struct(task: dict):
+    keys = task.keys()
+    required_keys = ["status", "desc"]
+    diff: set = required_keys - keys
+
+    if len(diff) > 0:
+        print(f"Error: {list(required_keys)} are required in data struct")
+        return True
+
+    if task["status"] not in ["in progress", "done", "not done"]:
+        print(f"Error: status must be 'in progress' or 'done'")
+        return True
+
+    if len(task["desc"]) == 0:
+        print(f"Error: desc should not be empty")
+        return True
+
+    return False
+
+def generate_task_id(task: dict):
+    task["id"] = uuid.uuid1().__str__().split('-')[0]
```

Comment:
Mesma coisa aqui em relação ao `get`

Reply:
Aqui também poderia ser feito da seguinte maneira:
```suggestion
    task["id"] = str(uuid.uuid1()).split('-')[0]
```

Acredito que assim ficaria mais "pythonic"

Reply:
> Mesma coisa aqui em relação ao `get`

aqui nao e possivel utilizar get pois estou atribuindo um valor e nao somente consultando